app/views/customers/index.html.erb

```diff
@@ -1,14 +1,39 @@
-<p style="color: green"><%= notice %></p>
-
 <h1>Customers</h1>
+<table>
+  <thead>
+    <tr>
+      <th>Full Name</th>
+      <th>Number of Orders</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @customers.each do |customer| %>
+      <tr>
+        <td><%= customer.full_name %></td>
+        <td><%= customer.orders.count %></td>
+        <td><%= link_to 'Show', customer_path(customer) %></td>
+        <td><%= link_to 'Edit', edit_customer_path(customer) %></td>
+        <td><%= link_to 'Destroy', customer_path(customer), method: :delete, data: { confirm: 'Are you sure?' } %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+
+# <h1>Customers</h1>
+
+# <div id="customers">
```

Comment:
Did you intend to comment these out? Even without any data, I'm seeing a lot of extraneous `#`'s on the page, because `#` isn't the right syntax for ERB comments.

![Screenshot 2024-09-07 at 10 27 39 PM](https://github.com/user-attachments/assets/8826bad2-8d99-4cbc-ad02-91e39f16fbd7)